### PR TITLE
update authz group task for 1.4

### PR DESCRIPTION
**Pull request by yangminzhu**
For https://github.com/istio/istio/issues/12394

### CODE REVIEW

File: content/en/docs/tasks/security/rbac-groups/index.md [hunk 1]
```diff
@@ -147,99 +147,62 @@ communicating with `httpbin`.
...
+workload if the requests are originated from specific groups.
 As there may be some delays due to caching and other propagation overhead,
-wait until the newly defined RBAC policy to take effect.
+wait until the newly defined authorization policy to take effect.
 
-1.  Enable the Istio RBAC for the namespace:
+1. Run the following command to create a `deny-all` policy in the `default` namespace.
+   The policy doesn't have a `selector` field, which applies the policy to every workload in the
+   `$NS` namespace. The `spec:` field of the policy has the empty value `{}`.
+   That value means that no traffic is permitted, effectively denying all requests.
```
Comment (lei-tang) [inline content/en/docs/tasks/security/rbac-groups/index.md]: Consider: s/That value/The empty value/

File: content/en/docs/tasks/security/rbac-groups/index.md [hunk 2]
```diff
@@ -26,17 +26,17 @@ To fulfill this prerequisite you can follow the Kubernetes
 
 ## Setup the required namespace and services
```
Comment (lei-tang) [inline content/en/docs/tasks/security/rbac-groups/index.md]: As the policy has been changed from service-oriented to workload-oriented, should "Setup the required namespace and services" be updated to "Setup the required namespace and workloads"? Similarly for the services mentioned in this section?

File: content/en/docs/tasks/security/rbac-groups/index.md [hunk 3]
```diff
@@ -147,99 +147,62 @@ communicating with `httpbin`.
...
     {{< text bash >}}
     $ kubectl exec $(kubectl get pod -l app=sleep -n $NS -o jsonpath={.items..metadata.name}) -c sleep -n $NS -- curl http://httpbin.$NS:8000/ip -s -o /dev/null -w "%{http_code}\n" --header "Authorization: Bearer $TOKEN"
     {{< /text >}}
 
-    Once the RBAC policy takes effect, the command returns the HTTP code 403.
+    Once the policy takes effect, the command returns the HTTP code 403.
 
-1.  To give read access to the `httpbin` service, create the `httpbin-viewer`
-service role:
+1.  To give read access to the `httpbin` workload, create the `httpbin-viewer`
```
Comment (lei-tang) [inline content/en/docs/tasks/security/rbac-groups/index.md]: To help readers understand better, consider adding an explanation of workload-selector based policy enforcement, e.g., replace "To give read access to the `httpbin` workload" with the following:
To give read access to the `httpbin` workload, which is selected through the label `app: httpbin`, ....

**Comment by yangminzhu** [comment]
@rcaballeromx @geeknoid Could you review this for the 1.4 release? It has been approved by the security team. Thanks.

**Pull request merged**